app/scripts/metamask-controller.js

```diff
@@ -962,6 +962,8 @@ export default class MetamaskController extends EventEmitter {
         'ExecutionService:unhandledError',
         'ExecutionService:outboundRequest',
         'ExecutionService:outboundResponse',
+        'SnapController:snapInstalled',
+        'SnapController:snapUpdated',
       ],
       allowedActions: [
```

Comment:
Although the function that is requiring the `SnapRegistry:update` action is never used at this point, I don't know if it's not a good idea to add `SnapRegistry:update` to the `allowedActions` ?

Reply:
If it's not used, I don't think we should add it?

Reply:
We can always add it in a future PR if it becomes necessary.